vuln/npm/423.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 423,
+  "title": "Command Injection - Generic",
+  "overview": "`fs-path` concatenates unsanitized input into exec()/execSync() commands",
+  "created_at": "2018-03-11",
+  "updated_at": "2018-05-11",
+  "publish_date": "2018-05-11",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "fs-path",
+  "cves": [],
+  "vulnerable_versions": "<=0.0.24",
```

Comment:
What's the rule here, btw? I have no reasons to believe that 0.0.25 would be not vulnerable (also: eh, no semver).

We have a large number of files (130, ~35% of the whole dataset),  that use
```js
"vulnerable_versions": "<=99.999.99999",
"patched_versions": "<0.0.0",
```
for this situation. Imo, we should use a single approach for those cases — either use that one here (and change if and when the package is patched), or use this one everywhere.

/cc @nodejs/security-wg thoughts?

Reply:
I feel that it is by default dooming all future releases to be vulnerable, and it's not like we are able to somehow keep track of every new release in every module we have a vulnerability before. In most cases where there isn't a fix, it is likely that the package is unmaintained and will probably not release a new update anyway.

Reply:
I have no strong opinion on this, but I think that the dataset should be consistent in that aspect.
Perhaps we should fix the old entries then?

Probably this should be filed as a separate issue, and this concern shouldn't block these PRs from landing.

Reply:
If you can shoot PRs for the existing issues that would be great.
As for the version top limit or not let's see if/what others have to say before I merge.